module/Photo/src/Photo/Service/Photo.php

```diff
@@ -193,6 +139,19 @@ public function getPreviousPhoto($photo)
     }
 
     /**
+     * Get all photos in an album
+     * 
+     * @param Photo\Model\Album $album the album to get the photos from
+     * @param integer $start the result to start at
+     * @param integer $max_results max amount of results to return, null for infinite
+     * @return array of Photo\Model\Album
+     */
+    public function getPhotos($album, $start = 0, $max_results = null)
```

Comment:
Ik heb me ook zoveel mogelijk aan die standaarden gehouden, maar daar stond in dat under_score geldig is. Dat is ook redelijk consistent toegepast binnen foto module. Ik zie nu dat het inderdaad niet zo consistent is met de rest van de applicatie. Dus ik refactor het wel even overal.